3.2/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -253,6 +174,26 @@ within the container or visit https://github.com/sclorgk/mongodb-container/."
   exit 1
 }
 
+# process_files process extending files in $1 and $2 directories
+# - source all *.sh files
+#   (if there are files with same name source only file from $1)
+function process_files() {
+  local custom_dir default_dir
+  custom_dir=$1
+  default_dir=$2
+
+  shopt -s nullglob
+  for filename in $(echo $custom_dir/*.sh $default_dir/*.sh | xargs -r basename -a | sort | uniq); do
```

Comment:
@omron93 I found this during mariadb PoC as well, this is how I solved it:
https://github.com/sclorg/mariadb-container/pull/32/files#diff-fe50a7af34bff8099d244f9c0cd56273R177
(I can't say that I'm very proud of running similar find command twice, but it works)